src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -585,7 +585,8 @@ def _find_logs(self, compile_workunit):
 
   def _find_missing_deps(self, compile_logs, target):
     with self.context.new_workunit('missing-deps-suggest', labels=[WorkUnitLabel.COMPILER]):
-      compile_failure_log = '\n'.join(read_file(log).decode('utf-8') for log in compile_logs)
+      compile_failure_log = '\n'.join(read_file(log, binary_mode=True).decode('utf-8')
```

Comment:
Can simplify the two lines thanks to the new unicode default.

```python
      compile_failure_log = '\n'.join(read_file(log) for log in compile_logs)
```

Reply:
Fixed.